specutils/tests/test_fitting.py

```diff
@@ -309,3 +310,77 @@ def test_fixed_parameters():
     g_fit = fit_lines(spectrum, g_init)
 
     assert g_fit.tied == g_init.tied
+
+
+def test_ignore_units():
+    """
+    Ignore the units
+    """
+
+    #
+    #  Ignore the units based on flag
+    #
+
+    # Create the spectrum
+    x_single, y_single = single_peak()
+    s_single = Spectrum1D(flux=y_single*u.Jy, spectral_axis=x_single*u.um)
+
+    # Fit the spectrum
+    g_init = models.Gaussian1D(amplitude=3.*u.Jy, mean=6.1*u.um, stddev=1.*u.um)
+    g_fit = fit_lines(s_single, g_init, ignore_units=True)
+    y_single_fit = g_fit(x_single)
+
+    # Comparing every 10th value.
+    y_single_fit_expected = np.array([3.69669474e-13, 3.57992454e-11, 2.36719426e-09, 1.06879318e-07,
+               3.29498310e-06, 6.93605383e-05, 9.96945607e-04, 9.78431032e-03,
+               6.55675141e-02, 3.00017760e-01, 9.37356842e-01, 1.99969007e+00,
+               2.91286375e+00, 2.89719280e+00, 1.96758892e+00, 9.12412206e-01,
+               2.88900005e-01, 6.24602556e-02, 9.22061121e-03, 9.29427266e-04])
+
+    assert np.allclose(y_single_fit[::10], y_single_fit_expected, atol=1e-5)
+
+
+    #
+    #  Ignore the units based on there not being units on the model
+    #
+
+    # Create the spectrum
+    x_single, y_single = single_peak()
+    s_single = Spectrum1D(flux=y_single*u.Jy, spectral_axis=x_single*u.um)
+
+    # Fit the spectrum
+    g_init = models.Gaussian1D(amplitude=3, mean=6.1, stddev=1.)
+    g_fit = fit_lines(s_single, g_init, ignore_units=True)
+    y_single_fit = g_fit(x_single)
```

Comment:
Oops! I hit the wrong button on my review and didn't include this inline comment.  This is the one I meant to leave...:

```suggestion
    y_single_fit = g_fit(s_single.spectral_axis)
    assert y_single_fit.unit == s_single.flux.unit
```

The above should be the goal.  That is, even if the *parameters* have no units, the input to the model should be in the spectral axis units of the spectrum, and the output should be the flux.